modules/weblog/collect.go

```diff
@@ -312,18 +312,32 @@ func (w *WebLog) collectCustomFields() {
 	if !w.line.hasCustomFields() {
 		return
 	}
+
 	for _, cv := range w.line.custom.values {
-		for _, pattern := range w.customFields[cv.name] {
-			if !pattern.MatchString(cv.value) {
-				continue
+		if patterns, ok := w.customFields[cv.name]; ok {
+			for _, pattern := range patterns {
+				if !pattern.MatchString(cv.value) {
+					continue
+				}
+				v, ok := w.mx.ReqCustomField[cv.name]
+				if !ok {
+					break
+				}
+				c, _ := v.GetP(pattern.name)
+				c.Inc()
+				break
 			}
-			v, ok := w.mx.ReqCustomField[cv.name]
+		} else if _, ok := w.customTimeFields[cv.name]; ok {
+			v, ok := w.mx.ReqCustomTimeField[cv.name]
 			if !ok {
 				break
 			}
-			c, _ := v.GetP(pattern.name)
-			c.Inc()
-			break
+			ctf, err := strconv.ParseFloat(cv.value, 64)
+			if err != nil || !isTimeValid(ctf) {
+				break
+			}
```

Comment:
oh, we need time multiplier

https://github.com/netdata/go.d.plugin/blob/a1c1fa72aba66e0f9dbf7ed9de239b849b47b474/modules/weblog/logline.go#L315-L319